.config/waybar/config

```diff
@@ -175,13 +175,13 @@
     "on-click": "dunstctl set-paused toggle",
     "restart-interval": 1
   },
-  "custom/pacman": {
-    "format": " {} update available",
-    "exec": "checkupdates | wc -l",
-    "exec-if": "test $(checkupdates | wc -l) -ne 0",
-    "interval": 3600,
-    "on-click": "checkupdates; pkill -SIGRTMIN+8 waybar"
-  },
+ "custom/pacman": {
```

Comment:
The indentation is incorrect.

If you fix it, I'll merge it.

Reply:
fixed